Lib/Search/ElasticSearch/Client.php

```diff
@@ -162,9 +163,34 @@ public function removeDocuments(ClassMetadata $class, array $documents)
      */
     public function removeAll(ClassMetadata $class, $query = null)
     {
-        $type = $this->getIndex($class->getIndexForRead())->getType($class->type);
+        $index = $class->getIndexForRead();
+        if ($class->timeSeriesField) {
+            // CRUD APIs are single-index APIs. See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs.html
```

Comment:
getIndexForRead() is a wildcard for timeseries, so the delete API doesn't work on wildcard?